services/client-api/models/board.js

```diff
@@ -0,0 +1,105 @@
+const { models } = require('../../../infrastructure/db');
+const logger = require('../../../infrastructure/logging/logger');
+const { getSequelize } = require('../../../infrastructure/db');
+
+/**
+* Finds all the boards of specific Team member
+* @param {*} req express request
+* @param {*} res express response
+*/
+const getBoards = async(req, res) => { //Not working yet
+    try {
+        const boards = await models.teammember.findByPk(req.teammemberId, { include : [
+            { model : models.board, as : 'boards', through: { attributes: [] }}
+        ], attributes: [] });
+        res.send(boards);
+    } catch(err) {
+        throw Error(err);
+    }
+};
+/*
+* Checks with a given boardId and given teammemberId if the user is a member in this board
+*/
+const isBoardMember = async(req, res, next) => {
+    try {
+        const sequelize = await getSequelize();
+        const isMember = await sequelize.query('SELECT * FROM board_teammember WHERE "boardId" = (:boardId) AND "teammemberId" = (:teammemberId)',{
```

Comment:
When using the try & catch block please notice you have only 1 await operation. When you have more than one async operation in your try block you are not catching the exceptions coming out of the async operations as needed.